tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -1656,6 +1703,39 @@ port::StatusOr<DeviceMemory<uint8>> CreateBatchNormBackwardWorkspace(
   }
   return workspace_allocator->AllocateBytes(workspace_size_in_bytes);
 }
+
+port::StatusOr<DeviceMemory<uint8>> CreateCtcLossWorkspace(
+    Stream* stream, const CudnnHandle& cudnn,
+    const CudnnCtcLossDescriptor& ctc_loss_desc,
+    const CudnnRnnStateTensorDescriptor& probs_desc,
+    const CudnnRnnStateTensorDescriptor& grads_desc,
+    const absl::Span<const int32>& labels_data,
```

Comment:
All `const absl::Span<const int32>&` should be `absl::Span<const int>`. Span acts like a pointer, so there is no need to pass by reference.